sympy/printing/ccode.py

```diff
@@ -320,6 +320,14 @@ def _print_Infinity(self, expr):
     def _print_NegativeInfinity(self, expr):
         return '-HUGE_VAL'
 
+    def _print_DiracDelta(self, expr):
+        return "static inline double DiracDelta (double %(s)s) \
+        { return %(s)s == 0 ? HUGE_VAL : 0; }" % {'s' : self._print(*expr.args)}
+
+    def _print_Heaviside(self, expr):
+        return "static inline double Heaviside (double %(s)s) \
```

Comment:
Why is this a function definition? Can this be used as an expression?

Reply:
Oh, I thought full function definition is to be printed. 
So should I remove `_print_DiracDelta` and change the other print function to return just an expression (or Heaviside is unnecessary too)?